decrypt_otpauth.py

```diff
@@ -167,20 +148,40 @@ def main(encrypted_otpauth_backup):
     data = data[:-data[-1]]
 
     # Decode wrapping archive
-    archive = DangerousUnarchive(data).top_object()
```

Comment:
If we remove the use of `DangerousUnarchive` here: Do we still need the definition of this class?

Reply:
unfortunately `DangerousUnarchive` is still needed for the next "layer" of archives. https://github.com/ewdurbin/decrypt-otpauth-files/blob/772221e84ca0c040502473f5a1ce2b840ac98df6/decrypt_otpauth.py#L163

Reply:
Ah, I'm sorry, I missed that.